src/django/api/signals.py

```
@@ -121,3 +123,34 @@ def set_origin_source_on_create(instance, created, **kwargs):
     FacilityLocation
 ]:
     post_save.connect(set_origin_source_on_create, sender=model)
+
+
+@receiver(post_save, sender=Source)
+def update_extended_fields_on_source_contributor_change(
+    sender, instance, created, update_fields, **kwargs
```

Ruff (0.13.2)

Unused function argument: `sender`

(ARG001)